openedx/core/djangolib/nose.py

```diff
@@ -1,30 +0,0 @@
-"""
-Utilities related to nose.
-"""
-import django_nose
-from django.core.management import call_command
-from django.db import DEFAULT_DB_ALIAS, connections, transaction
-
-
-class NoseTestSuiteRunner(django_nose.NoseTestSuiteRunner):
-    """Custom NoseTestSuiteRunner."""
```

Comment:
This is still referenced in cms.envs.test and lms.envs.test; the `TEST_RUNNER` override should probably just be removed from those.

Reply:
Done.